src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/OidCollection.cs

```diff
@@ -93,22 +73,10 @@ public void CopyTo(Oid[] array, int index)
             ((ICollection)this).CopyTo(array, index);
```

Comment:
Can you apply a change similar to https://github.com/dotnet/corefx/pull/7247/commits/a6353946a230f0281626439306b60690655053fb ?